pkgs/development/compilers/openjdk/meta.nix

```diff
@@ -5,7 +5,7 @@ lib: version: with lib; {
   maintainers = with maintainers; [ edwtjo asbachb ];
   platforms = [ "i686-linux" "x86_64-linux" "aarch64-linux" "armv7l-linux" "armv6l-linux" ];
   mainProgram = "java";
-  knownVulnerabilities = optionals (builtins.elem (versions.major version) [ "12" "13" "14" "15" "16" ]) [
+  knownVulnerabilities = optionals (builtins.elem (versions.major version) [ "12" "13" "14" "15" "16" "18" ]) [
```

Comment:
What about 17? (no, I am not a heavy java user.)

Reply:
I have opened the site. Strange that 18 is EOL but 17 isn't

Reply:
LTS releases.